maven-archetype-plugin/src/it/projects/ARCHETYPE-274_conditional_filesets/project.properties

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+archetypeGroupId=org.apache.maven.plugins.archetype.its
+archetypeArtifactId=archetype274-archetype
+archetypeVersion=1.0-SNAPSHOT
+
+groupId=com.company
+artifactId=myArtifactId
+version=1.0-SNAPSHOT
+package=com.company.project
+
+extraSupport=true
+anotherSupport=false
```

Comment:
missing new line at end

Reply:
I think we should consider adding something like. Either in parent pom or in this project

```xml
<pluginManagement>
      <plugins>
        <plugin>
          <groupId>com.diffplug.spotless</groupId>
          <artifactId>spotless-maven-plugin</artifactId>
          <configuration>
            <formats>
              <format>
                <includes>
                  <include>**.java</include>
                  <include>**pom.xml</include>
                  <include>**.txt</include>
                  <include>**.groovy</include>
                  <include>**.properties</include>
                </includes>
                <endWithNewline />
              </format>
            </formats>
          </configuration>
        </plugin>
    </pluginManagement>
 </plugins>
```